src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -35,6 +38,11 @@ public MarkdownV2Renderer(ParserMode mode, int maxSyntaxWidth)
             this._mode = mode;
         }
 
+        public MarkdownV2Renderer(int maxLineWidth)
```

Comment:
Actually, MarkdownV2Renderer not the right place for this logic.
MarkdownRenderer renders *something* as markdown, same goes for MamlRenderer.

We should create a separate class TextRenderer.